### PR TITLE
remove geocoding param

### PR DESCRIPTION
**Pull request by tjulien**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -286,18 +286,14 @@ class Radar {
       .catch(handleError(callback));
   }
 
-  static ipGeocode(arg0, arg1=defaultCallback) {
```
Comment (kochis) [inline src/index.js:289]: I think this is technically a breaking change. 

It's probably best to leave the initial arg in place, but do a `console.warn` in the case where `arg0` as an object is provided.
```js
console.warn('Radar SDK: ipGeocode parameters have been deprecated.');
```

Comment (tjulien) [reply src/index.js:289]: updated, can you take a look to see if it's what you had in mind?

Comment (kochis) [reply src/index.js:289]: Yep, looks good.

Comment (tjulien) [reply src/index.js:289]: Cool. Good to merge?

**Pull request merged**